api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/alerts_controller.rb

```diff
@@ -22,10 +22,10 @@ def create
 
     authorize! :manage_alert, kpi_alert
 
-    if (@alert = current_user.alerts.create(kpi_alert.attributes))
+    if (@alert = kpi_alert.save(kpi_alert.attributes))
```

Comment:
I agree with you @cesar-tonnoir that backward incompatibility is important to take care of.
However, usually, it is easier and faster to release new mno-hub than to release new mno-enterprise.
So maybe, we could just late for the mnohub updgrate to be released before doing the mno one.
I think it would be unfortunate to clutter the code if we can prevent it.

@ouranos What do you think ?